pkg/plugin/testhelper/pem.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testhelper provides commonly used test functions in different packages.
+package testhelper
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"testing"
+)
+
+// TestGeneratePrivateKey generates a rsa Key for testing use.
+// It returns the PEM decoded private key string and the rsa.PrivateKey it itself.
+func TestGeneratePrivateKey(tb testing.TB) (string, *rsa.PrivateKey) {
```

Comment:
If we're moving this into a dedicated package named "testutil", we can drop the `Test` prefix and call it `GeneratePrivateKey` instead.

Reply:
I also agree with @sqin2019 comment about including "rsa" in the name.